### PR TITLE
Add VLC in apps

### PR DESCRIPTION
**Pull request by Aperence**
VLC will support MPTCP starting from version 4.0

### CODE REVIEW

File: apps.md [hunk 1]
```diff
@@ -40,6 +40,12 @@ mentioned. Please note that other apps can be forced to use it by following
 | SSH | [OpenSSH-281.40.2](https://github.com/apple-oss-distributions/OpenSSH) | `applemultipath` option |
 | [Shellfish](https://secureshellfish.app) | ? | Settings / Advanced / 🌍 Multipath TCP |
 
+## iOS apps
+
+| Name | Version |
+| --- | --- |
+| [VLC](https://www.videolan.org/vlc/) | [v4.0](https://code.videolan.org/videolan/vlc-ios/-/commit/210c88b3e4e0dac0e4f2d18b3e3dfbe664693658) |
```
Comment (matttbe) [inline apps.md:47]: Can you add an "How to use" section please?

Will MPTCP be used by default? Or only if an option is enabled?

File: apps.md [hunk 2]
```diff
@@ -40,6 +40,12 @@ mentioned. Please note that other apps can be forced to use it by following
 | SSH | [OpenSSH-281.40.2](https://github.com/apple-oss-distributions/OpenSSH) | `applemultipath` option |
 | [Shellfish](https://secureshellfish.app) | ? | Settings / Advanced / 🌍 Multipath TCP |
 
+## iOS apps
+
+| Name | Version |
+| --- | --- |
```
Comment (matttbe) [inline apps.md:46]: (do you know if there are other apps we can add here? Might be good to add Siri, Apple Music, and Apple Maps in fact)

**Review by matttbe — CHANGES_REQUESTED** [review]
thanks! I have a few  questions if you don't mind.

File: apps.md (hunk 1 above)
Comment (Aperence) [reply apps.md:47]: It is enabled by default, I'll add this detail in the new section

Comment (Aperence) [reply apps.md:47]: (in handover mode)

File: apps.md (hunk 2 above)
Comment (Aperence) [reply apps.md:46]: Other than the apps published by Apple, I don't know any sadly

**Review by matttbe — APPROVED** [review]
thanks!

File: apps.md [hunk 3]
```diff
@@ -40,6 +40,15 @@ mentioned. Please note that other apps can be forced to use it by following
...
 | [Shellfish](https://secureshellfish.app) | ? | Settings / Advanced / 🌍 Multipath TCP |
 
+## iOS apps
+
+| Name | Version | How to use |
+| --- | --- | --- |
+| [VLC](https://www.videolan.org/vlc/) | [v4.0](https://code.videolan.org/videolan/vlc-ios/-/commit/210c88b3e4e0dac0e4f2d18b3e3dfbe664693658) | Enabled by default |
+| [Siri](https://www.apple.com/fr/siri/) | [>= iOS 7](https://www.tessares.net/apples-mptcp-story-so-far/) | Enabled by default |
+| [Apple Maps](https://www.apple.com/maps/) | [>= iOS 13](https://www.tessares.net/apple-pushes-multipath-tcp-further/) | Enabled by default |
+| [Apple Music](https://www.apple.com/music/) | [>= iOS 13](https://www.tessares.net/apple-pushes-multipath-tcp-further/) | Enabled by default |
```
Comment (matttbe) [inline apps.md:50]:
```suggestion
| [Apple Maps](https://www.apple.com/maps/) | [>= iOS 13](https://www.tessares.net/apple-pushes-multipath-tcp-further/) | Enabled by default |
| [Apple Music](https://www.apple.com/music/) | [>= iOS 13](https://www.tessares.net/apple-pushes-multipath-tcp-further/) | Enabled by default |
| [Siri](https://www.apple.com/fr/siri/) | [>= iOS 7](https://www.tessares.net/apples-mptcp-story-so-far/) | Enabled by default |
| [VLC](https://www.videolan.org/vlc/) | [v4.0](https://code.videolan.org/videolan/vlc-ios/-/commit/210c88b3e4e0dac0e4f2d18b3e3dfbe664693658) | Enabled by default |
```

By alphabetical order, like the other ones.

**Pull request merged**